Python/PyHSPlasma.pyi

```diff
@@ -0,0 +1,5616 @@
+# -*- coding: utf-8 -*-
+# copyage.py: Copy one age to another, with a new name and sequence prefix
+# Copyright (C) 2010 Branan Purvine-Riley
```

Comment:
Not quite :trollface:

Reply:
*Sigh* of course I copy from the file with this kind of nonsense...